src/components/dialog/dialog.js

```diff
@@ -536,6 +536,8 @@ function MdDialogDirective($$rAF, $mdTheming, $mdDialog) {
  *     `three` into the controller, with the value 3. If `bindToController` is true, they will be
  *     copied to the controller instead.
  *   - `bindToController` - `bool`: bind the locals to the controller, instead of passing them in.
+ *   - `resolve` - `{object=}`: Similar to locals, except it takes as values functions that return promises, and the	
```

Comment:
Let's change `{object=}` to `{function=}` to match the changes to the text.

Reply:
Oops. My Bad. Correcting it right away.